project/cookiecutter.json

```diff
@@ -13,6 +13,7 @@
   "language_code": ["en", "de", "es", "pt-br", "nl", "fi"],
   "github_organization": "collective",
   "container_registry": ["github", "docker_hub", "gitlab"],
+  "storage": ["relstorage", "zeo", "filestorage"],
```

Comment:
@tlotze Can we call it `devops_storage` or `zodb_storage` to be a bit more specific?

Reply:
True, `devops_storage` is more in line with the other variables referring to the deployment stack.